.github/workflows/publish_deriv_charts_and_update_deriv_app.yml

```diff
@@ -0,0 +1,126 @@
+name: Publish @deriv/deriv-charts and update package in deriv-app
+on:
+    workflow_dispatch:
+
+jobs:
+    build_and_publish_deriv_charts:
+        runs-on: Ubuntu-latest
+        permissions:
+            checks: write
+            contents: write
+        steps:
+            - name: Checkout SmartCharts
+              uses: 'binary-com/SmartCharts/.github/actions/checkout@master'
+              with:
+                  repository: 'binary-com/SmartCharts'
+                  path: SmartCharts
+                  ref: master
+
+            - name: Custom flutter-chart
+              id: flutter_chart
+              uses: 'binary-com/SmartCharts/.github/actions/checkout@master'
+              with:
+                  repository: 'regentmarkets/flutter-chart'
+                  path: flutter-chart
+                  ref: fe-changes
+                  token: ${{ secrets.REPO_READ_TOKEN }}
+
+            - uses: subosito/flutter-action@88cab4064b0a5145b0b53b42565b68248a3d7bcb
+              with:
+              flutter-version: '3.10.6'
+              channel: 'stable'
+              cache: true
+
+            - name: Add SSH key
+              run: |
+                  mkdir -p ~/.ssh
+                  echo "${{ secrets.SSH_KEY }}" > ~/.ssh/github_action_key
+                  chmod 600 ~/.ssh/github_action_key
+
+            - name: Build flutter
+              env:
+                  GIT_SSH_COMMAND: 'ssh -i ~/.ssh/github_action_key'
+              run: |
+                  cd SmartCharts/chart_app
+                  flutter pub get
+                  flutter build web --web-renderer html --release
+
+            - name: Setup node
+              uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8
+              with:
+                  node-version: 20.x
```

Comment:
### 
```suggestion
                  node-version: 18.x
```
Smarcharts still uses 18.x version

Reply:
Since smartcharts engine is using node >= 18, so 20 should be okay. also because semantic release doesnt work with node 18